src/css/components/po-button/po-button.css

```
@@ -6,117 +6,166 @@
   --font-button-font-sm {
     @apply --font-text-bold;
   }
-
-  --shadow-button-box-shadow: var(--shadow-button);
 }
 
 .po-button {
-  @apply --font-button-font;
-  background-color: var(--color-button-background-color);
-  border: solid 1px var(--color-button-border);
-  border-radius: 3px;
-  color: var(--color-button-color);
-  display: inline-block;
-  height: 44px;
-  padding: 0 16px;
-  cursor: pointer;
-  text-align: center;
-
-  &:hover {
-    background-color: var(--color-button-background-color-hover);
-    border: solid 1px var(--color-button-border-hover);
-    color: var(--color-button-color-hover);
-  }
+  font-family: var(--font-family-theme);
+  font-size: 1em;
+  font-weight: 700;
+  line-height: 100%;
+  border-radius: 0.25em;
+  border-width: 0.125em;
```

```suggestion
  border-width: 2px;
```

Conforme animalia-design-tokens, esse valor está em pixels.